hwdb/60-keyboard.hwdb

```diff
@@ -968,6 +968,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*300E[457]*:pvr
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*200E[45]*:pvr*
  KEYBOARD_KEY_ce=!                                      # Fn+F1  launch control setting
 
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*350V*:pvr*
+ KEYBOARD_KEY_ce=!prog1                                 # Fn+F1  Custom - launch control setting
+ KEYBOARD_KEY_d5=!wlan                                  # Fn+F12 WiFi On/Off
+ KEYBOARD_KEY_a0=!mute                                  # Fn+F6
+ KEYBOARD_KEY_ae=!volumedown                            # Fn+F7
+ KEYBOARD_KEY_b0=!volumeup                              # Fn+F8
```

Comment:
The others match the existing entry for # Series 7 / 9 can you just add the new ID there? Or are some of the assigned keys wrong there?